docs/recommendations.md

```diff
@@ -25,63 +27,157 @@ _diffrn_measurement.integration        stepwise
+The enumerated data item `_refine.diffraction_theory` can take the values 'dynamical' and 'kinematical', and should always be specified if refinement is performed using the dynamical theory. `_refine.diffraction_theory_details` is a free-text field where _e.g._ a description may be given of the parameters that were refined dynamically. 
 
-## `refine_diff.density_max`, `_min` and `_RMS`
+### `_refine_diff.potential_max`, `_min` and `_RMS`
 
-These quantities were defined in the original core CIF dictionary as indications of the final difference electron density. For electron diffraction, they should report the maximum, minimum and mean difference potential in the final difference Fourier map, and be expressed in units of electrons per ångström.
+These quantities are similar to those defined in the original core CIF dictionary describing features of the final difference electron density (_refine_diff.density_max` _etc._). For electron diffraction, they report the maximum, minimum and mean residual electrostatic potential in the final difference Fourier map, and are expressed in units of electrons per ångström.
 
-## `_refine_ls.abs_structure_z-score`
+### `_refine_ls.abs_structure_z-score`
 
 The statistical _z_ score can be determined as a robust measure of the confidence of an absolute structure assignment according to the procedure described in Klar, P. B., Krysiak, Y., Xu, H., Steciuk, G.,Cho, J., Zou, X. & Palatinus, L. (2023). Accurate structure models and absolute configuration determination using dynamical effects in continuous-rotation 3D electron diffraction data. _Nature Chem._ **15**, 848-855. This method is not specific to electron diffraction and may also be used for X-ray diffraction data, but for X-ray work it has largely been superseded by the Flack, Rogers and Parsons parameters.
+
+
+_Text below this line added 2024-07-22 in anticipation of new draft items and updated 2024-12-02_
+
+## More details of data collection
+
+Examples are based on (_a_) an unpublished CIF from *PETS2* describing a dynamical refinement data set 16-crot-030_dyn.cif_pets, and (_b_) the Zenodo deposition https://doi.org/10.5281/zenodo.7092770 of Hrushikesh & Suresh (2022).
+
+(_a_)
+
+```
+_exptl_crystal.mosaicity              0.050
+_exptl_crystal.method                 'From rocking curve, width 0.00070'
+```
+
+(_b_)
+```
+_diffrn_detector.ed_calibration_constant      0.008259
+_diffrn_source.convergence_angle              0.000(5)
+_diffrn_source.ed_diffracting_area_selection  probe
+_refine_ls.sample_thickness                   0.014
+_refine_ls.sample_shape_expression            'F(\t) = 1 - ( 1 - \t^2^)^1/2^'
+
+```
+
+### `_exptl_crystal.mosaicity` and `_exptl_crystal.mosaicity_method`
+
+The mosaicity (in degrees) and method of determination may be supplied. If known, the average mosaic block size in ångström units may also be given using `_exptl_crystal.mosaic_block_size`.
```

Comment:
The `_exptl_crystal.mosaic_method` data item is listed in the section title, but not explicitly described.

Reply:
My comment seems to be a false-positive. The first sentence does mention the "method of determination".